maps/xtehr2zibs2024/Procedure-NusringIntervention.md

```diff
@@ -133,7 +165,10 @@ On the other hand, the zib is more narrow in the sense that it only focusses on
 
 ### Comments
 
-The zib restricts the allowed terminology to two SNOMED refsets, whereas the Xt-EHR model prefers SNOMED. The terminology seems to align pretty well. (In the 2020 zib NIC is still present as a code system, but this has been deprecated as of 2024).
+The zib restricts the allowed terminology to two SNOMED refsets consisting of procedures as well as regime/therapy and situation (e.g. beoordeling van kennis over aandoening bij familie (situatie), debriefing van patiënt na fysieke inperking (regime/therapie)), whereas the Xt-EHR model prefers SNOMED.  
+(The terminology seems to align pretty well. (In the 2020 zib NIC is still present as a code system, but this has been deprecated as of 2024).)  
+It's expected that the SNOMED codes in the EHDSmodel will be restricted to just procedures. Therefore interventions about the situation and regime/therapy might be out of scope of the EHDSmodel.
```

Comment:
Maybe the conclusion is then that the terminologies don't align too well? You can probably conclude that all Xt-EHR codes will be valid in the zib, but that the zib envisions a broader use and that not all zib values will be valid in the Xt-EHR model.